File: codeforces/mashups/9/D.go
package main

import (
	"fmt"
	"sort"
)

// D. Jogo de cartas pra dois

func main() {

	var n int
	fmt.Scanf("%d", &n)

	a := make([]int64, n)
	b := make([]int64, 2)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &a[i])
	}

	sort.Slice(a, func(i, j int) bool { return a[i] > a[j] })

	for i := 0; i < n; i++ {
		b[i%2] += a[i]
	}

	fmt.Println(b[0]-b[1])
}

File: codeforces/mashups/9/D.cpp
#include <bits/stdc++.h>

using namespace std;

// D. Jogo de cartas pra dois

int main() {

    typedef long long ll;

	int n;
	cin >> n;

	vector<ll> a(n), b(2);

	for (int i = 0; i < n; i++) {
		cin >> a[i];
	}

	sort(a.begin(), a.end(), greater<ll>());

	for (int i = 0; i < n; i++) {
		b[i%2] += a[i];
	}

    //cout << a << "\n" << b << endl;

	cout << (b[0]-b[1]) << "\n";

    return 0;
}
